internal/builder/sandbox.go

```diff
@@ -80,7 +82,73 @@ const (
 	OverrideMiddleware MiddlewareName = "override"
 )
 
-func (sb *SandboxBuilder) AddOverrideMiddleware(worklaodPort int64, toLocal string, workloadNames ...string) *SandboxBuilder {
+type MiddlewareOverrideArg struct {
+	internal func(sb *SandboxBuilder, overrideName string) *models.SandboxesArgument
+	isSet    bool
+}
+
+func NewOverrideArgPolicy(defaultFallThroughStatus string, unimplementedResponseCodes []int) (*MiddlewareOverrideArg, error) {
+	policies := make(map[string]interface{})
```

Comment:
let's use the newer and more readable `any` instead of `interface{}` (they are the same thing just different names)